resources-build-logic/build.gradle.kts

```diff
@@ -12,6 +12,6 @@ repositories {
 dependencies {
     api("dev.icerock:mobile-multiplatform:0.13.0")
     api("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10")
-    api("com.android.tools.build:gradle:7.0.4")
+    api("com.android.tools.build:gradle:7.1.0")
     api("io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.15.0")
```

Comment:
use version catalog. and replace build-logic to moko-gradle-plugin

Reply:
I will resolve this comment in [other](https://github.com/icerockdev/moko-resources/pull/319) PR